xorgauth/management/commands/importaccounts.py

```diff
@@ -59,6 +59,29 @@ def handle(self, *args, **options):
                 if is_verbose:
                     print("Creating user %s (%d/%d)" % (hrid, idx_account + 1, accounts_num))
 
+            # Sometimes, display_name is empty. Use the first name or a name
+            # from the email address
+            if not account_data['display_name']:
+                if account_data['firstname']:
+                    if is_verbose:
+                        print("... using first name (%r) as display name" % account_data['firstname'])
+                    account_data['display_name'] = account_data['firstname']
+                else:
+                    # This should only happen for external accounts
+                    if account_data['type'] != 'xnet':
+                        raise CommandError("No display_name nor firstname for a non-external account")
```

Comment:
Nitpick : could be nice to dump `account_data` here in order to ease finding any faulty data in the future?